scalatest/src/main/scala/org/scalatest/tools/DashboardReporter.scala

```diff
@@ -513,8 +513,8 @@ private[scalatest] class DashboardReporter(directory: String,
   //
   def writeRunFile(event: Event, thisRunFile: File): Unit = {
     index = 0
-    var suiteRecord: SuiteRecord = null
-    val stack = new Stack[SuiteRecord]
+    var suiteRecord: SuiteRecord | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.